ignore/src/types.rs

```diff
@@ -161,7 +161,7 @@ const DEFAULT_TYPES: &'static [(&'static str, &'static [&'static str])] = &[
     ("js", &[
         "*.js", "*.jsx", "*.vue",
     ]),
-    ("json", &["*.json", "composer.lock"]),
+    ("json", &["*.json"]),
```

Comment:
I'm not so sure about this change. We also have `Cargo.lock` listed under the `toml` type, so this change is inconsistent with that. The thing is that we know `composer.lock` to be a JSON file, right? So it really should be searched when you ask to search for JSON files. Similarly for `Cargo.lock`, which we know to be TOML.

Maybe leave `composer.lock` here? We can still keep a separate overlapping `lock` type.

Reply:
I see, I will leave `composer.lock` with no change.